broker/config.py

```diff
@@ -54,6 +54,9 @@ def __init__(self):
         # https://docs.aws.amazon.com/Route53/latest/APIReference/API_AliasTarget.html
         self.CLOUDFRONT_HOSTED_ZONE_ID = "Z2FDTNDATAQYW2"
 
+        # the maximum from AWS is 20, and we alert when we have 18 on a given alb
+        self.MAX_CERTS_PER_ALB = 17
```

Comment:
I thought it was 25?

> Certificates per Application Load Balancer (excluding default certificates)	25

https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-limits.html

Reply:
good catch - updated the comment and value